src/test/kotlin/lotto/ManualTicketCountValidatorTest.kt

```diff
@@ -0,0 +1,25 @@
+package lotto
+
+import lotto.model.ManualTicket
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+import org.junit.jupiter.api.assertThrows
+
+class ManualTicketCountValidatorTest {
```

Comment:
```suggestion
class ManualTicketTest {
```
When refactoring the application logic, don't forget to apply the changes to tests as well!

Reply:
Totally missed that!